data-prepper-plugins/otel-trace-group-prepper/src/main/java/com/amazon/dataprepper/plugins/prepper/oteltracegroup/OtelTraceGroupPrepperConfig.java

```diff
@@ -0,0 +1,28 @@
+package com.amazon.dataprepper.plugins.prepper.oteltracegroup;
+
+import com.amazon.dataprepper.model.configuration.PluginSetting;
+import com.amazon.dataprepper.plugins.sink.elasticsearch.ConnectionConfiguration;
+import com.amazon.dataprepper.plugins.sink.elasticsearch.IndexConstants;
```

Comment:
Also thought upon the point, shared package is a reasonable option. Since the current use case for this plugin is to query from the same ingestion sink backend, I assume let it depend on the sink plugin is also defendable. If we deviate from this use case in the future, it might make more sense to create a shared library depends on need.